CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ More detailed release notes can be found on the [releases page](https://github.c
 
 * Go 1.17 or later is now required.
 * Icons for macOS bundles are now padded and rounded, disable with "-use-raw-icon"
+* Focus handling for List/Tree/Table are now at the parent widget not child elements
```

Comment:
This behavior should be added to the new GridWrap widget as well

Reply:
I agree. Likely just an oversight as it is such a new widget :)

Reply:
That's in now. Just needs some more tests.
Sorry I missed it - I was going by fyne_demo which it was not in, so I added it there too.